dist/js-joda.d.ts

```diff
@@ -1163,18 +1163,73 @@ declare namespace JSJoda {
         static from(temporal: TemporalAccessor): YearMonth
 
         static now(zoneIdOrClock?: ZoneId|Clock): YearMonth
+        static now0(): YearMonth
+        static nowZoneId(zone: ZoneId): YearMonth
+        static nowClock(clock: Clock): YearMonth
```

Comment:
these methods are our idea of "overloading" the `now` method. Since these are not part of the threeten/joda API,  we do not consider these functions as public in the API and i think they should thus not be defined in the typescript definitions.
Since this is not obvious to a user of js-joda not familiar with the threeten API, i think we will have to *also* mark these methods as `private` in the esdoc comments to make it more obvious.
And... to make it clear, you should *not* use these methods, but rather use the `now` method which will (depending on argument count and/or argument type delegate the call to the appropriate overloaded method. And i think the typescript definition of the `now` method allows you to pass any of the arguments of the overloaded methods (none for `now0`, ZoneId or Clock for `nowZoneId` and `nowClock`

Reply:
I wasn't aware of this. I will make the necessary changes later today.